setup.py

```diff
@@ -20,6 +20,6 @@
         "pyOpenSSL >= 19.1.0"
     ],
     extras_require={
-        "tests": ["timeout-decorator"],
+        "tests": ["pytest>=4.6.0,<5.1.2", "timeout-decorator", "funcy>=1.14,<2"],
```

Comment:
@Suor @MaxRis let's not do < version. It's better to fail fast and fix the discrepancy, then being not aware about new versions. We'll always have time to do this.